### ISSUE TITLE
Cache super-linter Docker image

### ISSUE THREAD

**Issue by bewuethr** [bewuethr/workflows#6]
Is there any way to cache the image locally? [This Q&A](https://stackoverflow.com/q/71180135/3266847) on Stack Overflow suggests using `docker save` and `docker load` together with the cache action.

**Comment by bewuethr** [comment]
Tried in #7; this seems fundamentally impossible, and even if it worked, it wouldn't be faster than downloading the image.

In this [workflow run](https://github.com/bewuethr/workflows/actions/runs/3510770433/jobs/5880896210), there is a cache hit:

![image](https://user-images.githubusercontent.com/8521043/202944342-7adce0cb-be5b-4acf-a45b-71d046e12abc.png)

The time to retrieve the cache (44s) and load the docker image from the tar file (40s) sums up to 1m 24s; this is more than the time it takes to download the image (1m 16s).

Aside from that, there doesn't seem to be a way to check the cache before the image is downloaded. The download happens before any step is run.

In other words: until this is natively supported by the runner, it doesn't seem possible to cache images for docker actions.
